File: Cursos/MasterLogicaProgramacion/EjerciciosPython/Ej3.py
def palabraEnFrase(palabra, frase):
    arrayFrase = frase.lower().split(" ")
    repeticiones = len([a for a in arrayFrase if a == palabra.lower()])

    return repeticiones

print(palabraEnFrase("hola", "Hola que hola tal hola como hola te hola va hola hola")); 

File: Cursos/MasterLogicaProgramacion/EjerciciosJavascript/Ej3.js
function palabraEnFrase(palabra, frase) {
    let arrayFrase = frase.toLowerCase().split(" ");
    let repeticiones = (arrayFrase.filter(x => x == palabra.toLowerCase())).length;

    return repeticiones;
}

console.log(palabraEnFrase("hola", "Hola que hola tal hola como hola te hola va hola hola"));